frontend/www/js/omegaup/components/group/Scoreboards.vue

```diff
@@ -0,0 +1,137 @@
+                  @blur="onGenerateAlias"
+                />
+              </label>
+            </div>
+
+            <div class="form-group col-md-6">
+              <label class="d-block">
+                {{ T.contestNewFormShortTitle_alias_ }}
+                <input :value="alias" class="form-control" disabled="true" />
+              </label>
+              <p class="help-block">
+                {{ T.contestNewFormShortTitle_alias_Desc }}
+              </p>
+            </div>
+          </div>
+          <div class="row">
+            <div class="form-group col-12">
+              <label class="d-block">
+                {{ T.groupNewFormDescription }}
+                <textarea
+                  v-model="description"
+                  rows="5"
+                  class="form-control"
+                ></textarea>
+              </label>
+            </div>
+          </div>
+
+          <button class="btn btn-primary" type="submit">
+            {{ T.groupEditScoreboardsAdd }}
+          </button>
+        </div>
+      </form>
+    </div>
+
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th>{{ T.groupEditScoreboards }}</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="scoreboard in scoreboards" :key="scoreboard.alias">
+          <td>
+            <a :href="`/group/${groupAlias}/scoreboard/${scoreboard.alias}/`">{{
+              scoreboard.name
+            }}</a>
+          </td>
+          <td>
+            <a
+              :href="`/group/${groupAlias}/scoreboard/${scoreboard.alias}/edit/`"
+              :title="T.wordsEdit"
+            >
+              <font-awesome-icon :icon="['fas', 'edit']" />
+            </a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import T from '../../lang';
+import latinize from 'latinize';
+import { types } from '../../api_types';
+
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+import { faEdit } from '@fortawesome/free-solid-svg-icons';
+library.add(faEdit);
+
+@Component({
+  components: {
+    FontAwesomeIcon,
+  },
+})
+export default class GroupScoreboards extends Vue {
+  @Prop() scoreboards!: types.GroupScoreboard[];
+  @Prop() groupAlias!: string;
+
+  T = T;
+  title: null | string = null;
+  alias: null | string = null;
+  description: null | string = null;
+
+  onGenerateAlias(): void {
+    if (this.title === null) {
+      return;
+    }
+
+    // Remove accents
+    let generatedAlias = latinize(this.title);
+
+    // Replace whitespace
+    generatedAlias = generatedAlias.replace(/\s+/g, '-');
+
+    // Remove invalid characters
+    generatedAlias = generatedAlias.replace(/[^a-zA-Z0-9_-]/g, '');
+
+    generatedAlias = generatedAlias.substring(0, 32);
+
+    this.alias = generatedAlias;
```

Comment:
nit: esto se puede hacer en una sola cadena:

```suggestion
    this.alias = latinize(this.title)  // Remove accents
        .replace(/\s+/g, '-')  // Replace whitespace
        .replace(/[^a-zA-Z0-9_-]/g, '')  // Remove invalid characters
        .substring(0, 32);
```

Reply:
Ohh, esto lo podemos reutilizar también en el componente de problemas. y en todos los lugares donde se genera un alias automáticamente. Gracias